pydis_site/apps/content/resources/guides/python-guides/windows/common-issues.md

```diff
@@ -0,0 +1,173 @@
+    > it. The result should be in the form `<PATH_TO_PYTHON>\python.exe`.
+    >
+    > Now compare that `PATH_TO_PYTHON` to the one from `pip -V`. If they're different, this is the cause of the
+    > `ModuleNotFoundError`. Follow the relevant "I'm running my code with ..." section below for how to fix this.
+
+    ##### I'm running my code with `python` from the terminal.
+
+    > If you are using a virtual environment, ensure you have activated it before running `pip install`. You can test
+    > this by running `pip -V` and checking the path is the one of your virtual environment.
+    >
+    > Alternatively, you can use the `py` command to specify the python version you want to pip install into, e.g.
+    > `py -3.9 -m pip install numpy`, and to specify the executable you want to run your code with, e.g.
+    > `py -3.9 my_script.py`.
+    >
+    > If your `pip` and `python` commands are referring to different python environments at the same time, it's possible
+    > your PATH is configured incorrectly. See our [guide for adding Python to PATH](../putting-python-on-path) for
+    > how to fix this.
+
+    ##### I'm running my code with the button in VSCode
+
+    > To run your code from the same environment you installed Python to, set the VSCode Python interpreter to the one
+    > at the path you found when running `pip -V` by following [this
+    > guide](https://code.visualstudio.com/docs/python/environments#_select-and-activate-an-environment) (basically
+    > `Ctrl+Shift+P -> Python: Select Interpreter -> select the one desired`). The path should be in the form
+    > `PATH_TO_PYTHON\python.exe`
+    >
+    > Alternatively, you can pip install to the currently activated environment. If you have configured VSCode to use a
+    > virtual environment, it should automatically activate it when you open a new terminal, so running the
+    > `pip install` command in the VSCode terminal should work. If you are not using a virtual environment, you can use
+    > the `py` launcher to specify the installation you want to install to.
+
+### I `pip` installed a command line program but it isn't recognized
+
+You pip installed a command line program like [`pyinstaller`](https://pypi.org/project/pyinstaller/) or
+[`black`](https://pypi.org/project/black/) that is intended to be run from the command line, but it does not work. Two
+common causes for this are:
+
+-   You are using Python from the Microsoft Store ([which we don't recommend](../miscrosoft-store)).
+
+    To check if this is the case, type `pip -V`. If the path output includes something like
+    `PythonSoftwareFoundation.Python.3.10_3.10.2288.0_x64__qbz5n2kfra8p0`, you are using Python from the Microsoft
+    Store. To fix this you will either have to use the full path to the scripts you want to use, or [uninstall the
+    Microsoft Store Python](/static/images/content/python-on-windows/ms_store_uninstall.png) and [install Python
```

Comment:
would it be better to link to the uninstall section on the page instead here?

Reply:
Good ideal. I'll make `microsoft-store/#uninstalling` a section and change this link.